Pod/Classes/WPMediaPickerOptions.h

```diff
@@ -0,0 +1,36 @@
+#import <Foundation/Foundation.h>
+#import "WPMediaCollectionDataSource.h"
+
+@interface WPMediaPickerOptions: NSObject
+
+/**
+ If set the picker will show a cell that allows capture of new media, that can be used immediatelly
+ */
+@property (nonatomic, assign) BOOL allowCaptureOfMedia;
```

Comment:
I wonder if we should rename this to something like `showCaptureCell` or `showCameraPreview` or something.